testsuite/features/secondary/srv_payg_ssh_connection.feature

```diff
@@ -0,0 +1,124 @@
+    When I follow the left menu "Admin > Setup Wizard > Pay-as-you-go"
+    And I follow "my-host-full.local"
+    And I click on "Edit Information"
+    And I enter "My new Full Instance" as "description"
+    And I click on "Save" in "Information" modal
+    Then I should see a "Pay-as-you-go properties updated successfully" text
+    And I should see a "My new Full Instance" text in element "Info-panel-wrapper"
+
+  Scenario: Edit instance ssh connection data
+    When I follow the left menu "Admin > Setup Wizard > Pay-as-you-go"
+    And I follow "my-host-full.local"
+    And I click on "Edit Instance"
+    And I enter "221" as "port"
+    And I enter "NewRootFull" as "username"
+    And I enter "NewPasswordFull" as "password"
+    And I enter "newKeyFull" as "key"
+    And I enter "newKeyPasswordFull" as "key_password"
+    And I click on "Save" in "Instance SSH connection" modal
+    Then I should see a "Pay-as-you-go properties updated successfully" text
+    And I should see a "221" text in element "Instance-panel-wrapper"
+    And I should see a "NewRootFull" text in element "Instance-panel-wrapper"
+    And I should see a "NewPasswordFull" text in element "Instance-panel-wrapper"
+    And I should see a "newKeyFull" text in element "Instance-panel-wrapper"
+    And I should see a "newKeyPasswordFull" text in element "Instance-panel-wrapper"
+
+  Scenario: Edit bastion ssh connection data
+    When I follow the left menu "Admin > Setup Wizard > Pay-as-you-go"
+    And I follow "my-host-full.local"
+    And I click on "Edit Bastion"
+    And I enter "my-new-bastion.local" as "bastion_host"
+    And I enter "222" as "bastion_port"
+    And I enter "b_new_rootFull" as "bastion_username"
+    And I enter "b_new_passwordFull" as "bastion_password"
+    And I enter "b_new_keyFull" as "bastion_key"
+    And I enter "b_new_keyPasswordFull" as "bastion_key_password"
+    And I click on "Save" in "Bastion SSH connection" modal
+    Then I should see a "Pay-as-you-go properties updated successfully" text
+    And I should see a "my-new-bastion.local" text in element "Bastion-panel-wrapper"
+    And I should see a "222" text in element "Bastion-panel-wrapper"
+    And I should see a "b_new_rootFull" text in element "Bastion-panel-wrapper"
+    And I should see a "b_new_passwordFull" text in element "Bastion-panel-wrapper"
+    And I should see a "b_new_keyFull" text in element "Bastion-panel-wrapper"
+    And I should see a "b_new_keyPasswordFull" text in element "Bastion-panel-wrapper"
+
+  Scenario: Delete minimal information for payg ssh connection data
+    When I follow the left menu "Admin > Setup Wizard > Pay-as-you-go"
+    And I follow "my-host.local"
+    And I click on "Delete"
+    And I click on "Delete" in "Delete Pay-as-you-go" modal
+    Then I should not see a "my-host.local" link
+    And I should not see a "my-host.local" text
+
+  Scenario: Delete Full information for payg ssh connection data
```

Comment:
Prefix with `Cleanup:` please (unless it's not part of the idempotency cleanup)

Reply:
the delete is part of the test, it's not just for cleanup